### PR TITLE
Improve UI of Manage Courses

### PR DESCRIPTION
**Pull request by ajoxales**

### CODE REVIEW

**Review by jaymercado — CHANGES_REQUESTED** [review]
Why does your code include my commit? It should only include your commit (code relevant to your ticket)

File: src/app/(views)/managed-courses/[courseId]/modules/[moduleId]/edit/page.tsx [hunk 1]
```diff
@@ -75,38 +80,110 @@ export default function EditModule() {
...
-        {errors.type && <CFormText className="text-danger">This field is required</CFormText>}
-      </CInputGroup>
-
-      <ModuleContentInput type={watch('type')} value={watch('content')} setValue={setValue} />
-
-      <CButton type="submit" color="primary" disabled={updatingModule}>
-        {updatingModule ? <CSpinner size="sm" /> : 'Save'}
-      </CButton>
-    </CForm>
+    <>
```
Comment (jaymercado) [inline src/app/(views)/managed-courses/[courseId]/modules/[moduleId]/edit/page.tsx]: Do we need this commented code? if yes, put a comment above explaining why this is still here.

File: src/app/(views)/managed-courses/[courseId]/modules/[moduleId]/page.tsx [hunk 2]
```diff
@@ -37,22 +37,24 @@ export default function Module() {
...
-      </button>
-      <section>
-        <p>ID: {courseModule.id}</p>
-        <p>Title: {courseModule.title}</p>
-        <p>Description: {courseModule.description}</p>
-        <p>Type: {courseModule.type}</p>
-        <p>Content: {courseModule.content}</p>
-      </section>
-    </div>
+    <>
```
Comment (jaymercado) [inline src/app/(views)/managed-courses/[courseId]/modules/[moduleId]/page.tsx]: Why the empty tag if there's only element inside anyways?

File: src/app/(views)/managed-courses/[courseId]/modules/create/page.tsx [hunk 3]
```diff
@@ -55,38 +60,110 @@ export default function CreateModule() {
...
-        {errors.title && <CFormText className="text-danger">This field is required</CFormText>}
-      </CInputGroup>
-      <CInputGroup>
-        <CFormLabel htmlFor="description">Description</CFormLabel>
-        <CFormInput id="description" {...register('description', { required: true })} />
-        {errors.description && (
-          <CFormText className="text-danger">This field is required</CFormText>
-        )}
-      </CInputGroup>
+    <>
```
Comment (jaymercado) [inline src/app/(views)/managed-courses/[courseId]/modules/create/page.tsx]: Do we need this commented code? if yes, put a comment above explaining why this is still here.

File: src/app/(views)/managed-courses/[courseId]/modules/page.tsx [hunk 4]
```diff
@@ -16,7 +39,8 @@ export default function Modules() {
   }
 
   return (
-    <table>
+    <>
+      {/* <table>
```
Comment (jaymercado) [inline src/app/(views)/managed-courses/[courseId]/modules/page.tsx]: Do we need this commented code? if yes, put a comment above explaining why this is still here.

File: src/app/(views)/managed-courses/[courseId]/modules/page.tsx [hunk 5]
```diff
@@ -51,6 +75,83 @@ export default function Modules() {
...
+                  <CTableHeaderCell>
+                    <small>Type</small>
+                  </CTableHeaderCell>
+                  <CTableHeaderCell>
+                    <small>Actions</small>
+                  </CTableHeaderCell>
+                </CTableRow>
+              </CTableHead>
+              <CTableBody>
+                {courses[0]?.modules.map((module) => (
```
Comment (jaymercado) [inline src/app/(views)/managed-courses/[courseId]/modules/page.tsx]: If there are no modules, add a table row saying "No modules" or something along the lines thereof.

File: src/app/(views)/managed-courses/[courseId]/page.tsx [hunk 6]
```diff
@@ -61,66 +103,223 @@ export default function Course() {
...
-              </td>
-              <td>{module.title}</td>
-              <td>{module.type}</td>
-              <td>
-                <Link href={`/managed-courses/${course.id}/modules/${module.id}/edit`}>Edit</Link>
-                <button
-                  type="button"
-                  onClick={() => deleteModule(module.id)}
-                  disabled={deletingModule}
+    // <div>
```
Comment (jaymercado) [inline src/app/(views)/managed-courses/[courseId]/page.tsx]: Do we need this commented code? if yes, put a comment above explaining why this is still here.

File: src/app/(views)/managed-courses/[courseId]/page.tsx [hunk 7]
```diff
@@ -61,66 +103,223 @@ export default function Course() {
...
+          <CCardBody>
+            <CRow>
+              <CCol>
+                <CBadge color="primary" shape="rounded-pill" className="text-normal mb-1">
+                  Course ID: {courseId}
+                </CBadge>
+                <CCardTitle style={{ fontSize: '1.5rem', fontWeight: 600 }}>
+                  {course.title}
+                </CCardTitle>
+                {/* <CCardText>Course ID: {courseId}</CCardText> */}
```
Comment (jaymercado) [inline src/app/(views)/managed-courses/[courseId]/page.tsx]: Do we need this commented code? if yes, put a comment above explaining why this is still here.

File: src/app/(views)/managed-courses/page.tsx [hunk 8]
```diff
@@ -41,62 +53,112 @@ export default function ManagedCourses() {
...
+                        <div className="d-flex justify-content-end mt-auto">
+                          <Link
+                            href={`/managed-courses/${course.id}`}
+                            className="me-2 d-flex align-items-center gap-1"
+                            style={{ textDecoration: 'none' }}
+                          >
+                            <small className="text-secondary d-none d-sm-inline">View Course</small>
+                            <i className="bi bi-chevron-right text-secondary"></i>
+                          </Link>
+                          {/* <CDropdown alignment="end">
```
Comment (jaymercado) [inline src/app/(views)/managed-courses/page.tsx]: Do we need this commented code? if yes, put a comment above explaining why this is still here.

**Review by jaymercado — CHANGES_REQUESTED** [review]
Why not write a reusable modal at this point?

**Review by jaymercado — CHANGES_REQUESTED** [review]
Add Loom video and record a walkthrough

**Comment by ajoxales** [comment]
https://www.loom.com/share/0ab2b426eaca493284497ff1d0f1af6e?sid=4e3cc2d2-1364-468b-916f-6d69633d86a7

**Pull request closed without merging**